### DISCUSSION TITLE
Getting Command 'kolibri' not found, in ubuntu

### DISCUSSION THREAD

**Discussion by muditchoudhary** [learningequality/kolibri#9750] [Q&A] [answered]
Hello I was setting up the Kolibri locally and got this error

![image](https://user-images.githubusercontent.com/74391865/192800973-ea50757b-c749-4268-af68-72865aa72d18.png)

 while running this command 
`kolibri manage migrate`

Can someone help to resolve this?

OS: Ubuntu 20.04
Terminal: Bash default by Ubuntu

P.S. What other details do I need to share please tell me, I'm new here.

Thanks!! :)

**Comment by jredrejo** [comment]
hi @muditchoudhary , how did you install kolibri? Seeing your prompt you seem to be using a virtenv configuration. If that's the case you have to be sure  you've installed it to be accesible from the venv

**Reply by muditchoudhary** [reply]
Yes I did setup the venv as mentioned in the prerequisite docs and installed the py-env and python 9.8.x version. 

My question how can I install it ( I thinks you are talking about kolibri) to be accessible from venv. I did not found any instructions regarding this in docs.

**Reply by MisRob** [reply, marked as answer]
`kolibri` command should be made accessible by `pip install -e .` step of [Installing Python dependencies](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#install-python-dependencies)

**Reply by MisRob** [reply]
Apologies for the late reply, in the past, we weren't using GitHub Discussion very actively and missed the message.

**Reply by muditchoudhary** [reply]
Thanks!! it worked. 

I missed "Dot" at the end of this command: `pip install -e .`
